articles/ros2_dds_security_pkcs11_support.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: ROS2 DDS security PKCS#11 URI support
+abstract:
+  The [DDS-Security specification][dds_security] defines the use of Hardware Security Modules (HSM) and PKCS#11 URIs as an alternative to private keys and certificates stored in the file system. Current implementation only supports these tokens to be directly stored in the file system as `.pem` files. This is a design proposal to support PKCS#11 URIs.
```

Comment:
one sentence per line, and so is everywhere else. (if i am not mistaken, this is required format for doc.)

```suggestion
  The [DDS-Security specification][dds_security] defines the use of Hardware Security Modules (HSM) and PKCS#11 URIs as an alternative to private keys and certificates stored in the file system.
  Current implementation only supports these tokens to be directly stored in the file system as `.pem` files.
  This is a design proposal to support PKCS#11 URIs.
```